source/data-api/authenticate.txt

```diff
@@ -0,0 +1,160 @@
+.. _data-api-authenticate-requests:
+
+==============================
+Authenticate Data API Requests
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Data API endpoints generally require that incoming requests include
+authentication information for the user calling the endpoint. This lets
+the endpoint enforce rules and validate document schemas for each
+request.
+
+Users authenticate requests by providing authentication information in
+the request headers. They can either use an authentication access token
+or directly include login credentials from any of the following
+authentication providers:
+
+- :ref:`Email/Password <email-password-authentication>`
+
+- :ref:`API Key <api-key-authentication>`
+
+- :ref:`Custom JWT <custom-jwt-authentication>`
+
+.. note::
+   
+   You must :ref:`enable an authentication provider
+   <authentication-providers>` before users can authenticate with
+   it.
+
+.. _data-api-bearer-authentication:
+
+Bearer Authentication
+---------------------
+
+The Data API supports Bearer Authentication, which lets you authenticate
+a request by including a valid user access token in the request's
```

Comment:
would be good to have a little list like what you have on [line 78](https://github.com/mongodb/docs-app-services/pull/216/files#diff-dbdb57fdc4ec9bb1dda1f570b88cb482af2271f24f1f819a08c69abaf7ef1cdaR79), but for why you'd use Bearer auth. 

this would be client-side use cases (or really anywhere you'd use token-based auth), including browser and mobile apps.